df_chat/tests/test_chat.py

```diff
@@ -247,6 +249,88 @@ async def test_chat_multiple_rooms(self):
         await communicator2.disconnect()
         await communicator3.disconnect()
 
+    def test_create_message(self):
+        User = get_user_model()
+        user = User.objects.create_superuser(username="test_username", password="test_password")
```

Comment:
Please reuse the "TestChat.create_user" method, to create a user.

Reply:
Done!